GE07-TEAM6/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, Command
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _action_done(self):
+        res = super()._action_done()
+
+        if res:
```

Comment:
This check is unnecessary, this always returns True [_action_done definition](https://github.com/odoo/odoo/blob/0b30a4e54b79a8f52d2330ccdda487113126c6ee/addons/stock/models/stock_picking.py#L902)